kyo-bench/src/main/scala/kyo/bench/HttpClientBench.scala

```diff
@@ -4,6 +4,8 @@ import org.http4s.ember.client.EmberClientBuilder
 
 class HttpClientBench extends Bench.ForkOnly("pong"):
 
+    override val zioRuntimeLayer = super.zioRuntimeLayer.merge(zio.http.Client.default)
```

Comment:
theoretically I think we could make the parent class do this, but we would need to make Bench accept an `R` parameter. Added to a ZEnvironment will need a tag for the type representation.

Let me know what you think @fwbrasil